winlogbeat/checkpoint/file_windows.go

```diff
@@ -29,8 +30,8 @@ func createWriteThroughFile(path string) (*os.File, error) {
 		syscall.GENERIC_READ|syscall.GENERIC_WRITE,               // Access Mode
 		uint32(syscall.FILE_SHARE_READ|syscall.FILE_SHARE_WRITE), // Share Mode
 		nil, // Security Attributes
-		syscall.CREATE_ALWAYS,                                          // Create Mode
-		uint32(syscall.FILE_ATTRIBUTE_NORMAL|_FILE_FLAG_WRITE_THROUGH), // Flags and Attributes
+		syscall.CREATE_ALWAYS, // Create Mode
+		uint32(syscall.FILE_ATTRIBUTE_NORMAL|_FILE_FLAG_WRITE_THROUGH|_FILE_FLAG_NO_BUFFERING), // Flags and Attributes
```

Comment:
We could move this file creation code up to libbeat so that we could share it. That would be simpler than trying to unify the registrar (which is still a good idea, but more time consuming).